update.sh

```diff
@@ -30,7 +30,7 @@ declare -A base=(
 )
 
 declare -A extras=(
-	[apache]='\&\& a2enmod headers remoteip ;\\\n    {\\\n      echo RemoteIPHeader X-Real-IP ;\\\n      echo RemoteIPTrustedProxy 10.0.0.0/8 ;\\\n      echo RemoteIPTrustedProxy 172.16.0.0/12 ;\\\n      echo RemoteIPTrustedProxy 192.168.0.0/16 ;\\\n    } > /etc/apache2/conf-available/remoteip.conf;\\\n    a2enconf remoteip'
+	[apache]='a2enmod headers remoteip ;\\\n    {\\\n      echo RemoteIPHeader X-Real-IP ;\\\n      echo RemoteIPTrustedProxy 10.0.0.0/8 ;\\\n      echo RemoteIPTrustedProxy 172.16.0.0/12 ;\\\n      echo RemoteIPTrustedProxy 192.168.0.0/16 ;\\\n    } > /etc/apache2/conf-available/remoteip.conf;\\\n    a2enconf remoteip;'
```

Comment:
Why is there  a ";" at the end of line ? Is not syntax error.

Reply:
> Why is there a ";" at the end of line ? Is not syntax error.

@Kaochkidu that's not a mistake. This is to mark the end of the command.
This code will insert those "_extra_" commands in the Dockerfile `RUN` section for Apache, so we need to make sure the command ends before executing next command.